### PR TITLE
Added fixtures

### PR DESCRIPTION
**Pull request by vlad-bruma**

### CODE REVIEW

File: src/DataFixtures/JobFixtures.php [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+     *
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager): void
+    {
+        $faker = Factory::create();
+
+        for ($i = 0; $i < 12; $i++) {
+            $job = new Job();
+            $job->setType($faker->shuffleArray(['private', 'public'])[0]);
```
Comment (gregurco) [inline src/DataFixtures/JobFixtures.php]: use `$faker->randomElement()`

File: src/DataFixtures/JobFixtures.php [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            $job->setCompany($faker->company);
+            $job->setLogo($job->getCompany());
+            $job->setUrl($faker->url);
+            $job->setPosition($faker->jobTitle);
+            $job->setLocation($faker->city);
+            $job->setDescription($faker->text);
+            $job->setHowToApply($faker->text);
+            $job->setToken($faker->md5);
+            $job->setPublic($faker->shuffleArray([true, false])[0]);
```
Comment (gregurco) [inline src/DataFixtures/JobFixtures.php]: also randomElement

File: src/Entity/Affiliate.php [hunk 3]
```diff
@@ -207,4 +208,16 @@ public function removeCategory(Category $category): self
 
         return $this;
     }
+
+    /**
+     * @ORM\PrePersist
+     *
+     * @return void
+     */
+    public function setTimestamp(): void
```
Comment (gregurco) [inline src/Entity/Affiliate.php]: better to name this method as the action when it's triggered: `public function prePersist(): void`

Comment (gregurco) [reply src/Entity/Affiliate.php]: and in this case method will not look like setter

File: src/Traits/Timestamps.php [hunk 4]
```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace App\Traits;
+
+use Doctrine\ORM\Mapping as ORM;
+
+trait Timestamps
```
Comment (gregurco) [inline src/Traits/Timestamps.php]: if it's trait, then better add suffix Trait: `trait TimestampsTrait`

**Review by gregurco — CHANGES_REQUESTED** [review]

**Comment by vlad-bruma** [comment]
Added adjustments

File: src/DataFixtures/AffiliateFixtures.php [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+        $manager->flush();
+    }
+
+    /**
+     * @return array
+     */
+    public function getDependencies(): array
+    {
+        return array(
```
Comment (gregurco) [inline src/DataFixtures/AffiliateFixtures.php]: use `[]` instead of `array()`

File: src/DataFixtures/JobFixtures.php [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+
+        $manager->flush();
+    }
+
+    /**
+     * @return array
+     */
+    public function getDependencies(): array
+    {
+        return array(
```
Comment (gregurco) [inline src/DataFixtures/JobFixtures.php]: also `[]` array

**Review by gregurco — CHANGES_REQUESTED** [review]

**Pull request merged**